pubspec.yaml

```diff
@@ -7,26 +7,26 @@ homepage: https://github.com/jhomlala/betterplayer
 documentation: https://jhomlala.github.io/betterplayer/
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=1.10.0"
+  sdk: ">=3.4.3 <4.0.0"
+  flutter: "3.22.2"
 
 dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.3
-  wakelock: ^0.5.2
+  wakelock: ^0.6.2
```

Comment:
wakelock got replaced by wakelock_plus

Reply:
Yep, I found out this, but I didn’t have time today to fix it. Big thank you for your comment :)

Reply:
Fixed :)